File: Beecrowd/1051 Imposto de Renda/main.py
X = float(input())
	
if X >= 0 and X <= 2000:
  print("Isento")

elif X <= 3000:
  imposto = (X - 2000) * 0.08
  print("R$ %0.2f" %imposto)

elif X <= 4500:
  imposto = (X - 3000) * 0.18 + 80
  print("R$ %0.2f" %imposto)

else:
  imposto = (X - 4500) * 0.28 + 350
  print("R$ %0.2f" %imposto)

File: Beecrowd/1051 Imposto de Renda/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	double X, imposto;
	
	cin >> X;
	cout << fixed;
	cout.precision(2);
	if(X >= 0.0 && X <= 2000.0)
		cout << "Isento" << endl;
	else if(X <= 3000.0){
		imposto = (X - 2000) * 0.08;
		cout << "R$ " << imposto << endl;
	}
	else if(X <= 4500.0){
		imposto = (X - 3000) * 0.18 + 80;
		cout << "R$ " << imposto << endl;
	}
	else{
		imposto = (X - 4500) * 0.28 + 350;
		cout << "R$ " << imposto << endl;
	}
 
    return 0;
}